src/pages/CreateCourse.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from "react";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+import { useCookies } from "react-cookie";
+import moment from "moment";
+
+import { Button, Input, Layout, InputNumber, Typography, DatePicker } from "antd";
+
+import Header from "../components/Header";
+import "../styles/CreateCourse.css";
+
+const CreateCourse: React.FC<RouteComponentProps> = (props) => {
+  const [ cookies ] = useCookies(['accessToken']);
+  let [ courseName, setCourseName ] = useState("");
+  let [ role, setRole ] = useState("");
+  let [ startYear, setStartYear ] = useState<Number | undefined>(undefined);
+  let [ courseSize, setCourseSize ] = useState<String | Number | undefined>(undefined);
+
+  useEffect(() => {
+    if (!cookies['accessToken']) {
+      props.history.push("/Login");
+    }
+  }, []);
+
+  const onConfirmClick = () => {
+    // TODO: Write code here to create new course (post request)
+    props.history.push("/Dashboard");
+  };
+
+  const onCancelClick = () => {
```

Comment:
@PreetPatel isn't it safer to put push("/Dashboard") in case the user decides to type the URL straight in instead of following the app flow? I can change it if you really want me to though